stories/DropdownLight/DropdownLight.regression-test.story.jsx

```diff
@@ -0,0 +1,16 @@
+import DropdownLight from '../../components/DropdownLight';
+import { itemsMock } from './mock';
```

Comment:
how about of renders the two ways of using this component?
the two types of items: the string way and the object way.
remember the spike...
This method only tests the object way...
i suggest a **itemsStringMock** and **itemsObjectMock** names.